function_examples/token-hider/README.md

```diff
@@ -17,24 +17,27 @@ When you deploy your site to Netlify, you'll need to define the following [build
 ## Localhost
 
 ### 1. Installation
-
 Clone the repo, from the command line, go to `function_examples/token-hider` folder and run `npm install`.
 
 ### 2. Run Lambda Functions
-
-Type in the following script in your console:
 `API_URL="https://maps.googleapis.com/maps/api/geocode/json" API_TOKEN="your Google Maps API key" npm run lambda:dev`
+
 This will start a local dev server. Requests to `http://localhost:9000/getapi?address=<the address>` will now be handled by `functions/getapi.js`.
 e.g. try this URL in your browser:
 `localhost:9000/getapi?address=220+Stanhope+St,+Brooklyn,+NY`
 
 #### Proxying for local development
-
 Examine the `server.js` file. It is already configured to proxy all requests destined for `/.netlify/functions` to the server running on port 9000.
 
 ### 3. Test the web site
-
 Open a new console and type in: `npm run site:dev`
 The site is now available at `http://localhost:1234/`
 
-Open the site in your browser and verify that the API key is not anywhere on the client side.
+_Notes: Make sure you run Functions first before you run the site._
```

Comment:
This is included in step 2 (line 23), but something weird with formatting combined the command with some variables. When Step 2 is fixed, I don't think this line will be necessary.